README.md

```diff
@@ -103,26 +103,27 @@ needed.
    hostname mptest-1.comdb2.example.com
    ```
 
-7. On one machine, create a database.
+7. On one machine, create a database - this example creates a database called _mikedb_ stored in _~/db_.
+
    ```
-   comdb2 --create --dir /home/mponomar/db mikedb
+   comdb2 --create --dir ~/db mikedb
    ```
    
 8. Configure the nodes in the cluster:
    ```
-   vi /home/mponomar/db/mikedb.lrl
+   vi ~/db/mikedb.lrl
    add
    cluster nodes mptest-1.comdb2.example.com mptest-2.comdb2.example.com
    ```
    
 9. On other nodes, copy the database over:
    ```
-   copycomdb2 mptest-1.comdb2.example.com:/home/mponomar/db/mikedb.lrl
+   copycomdb2 mptest-1.comdb2.example.com:~/db/mikedb.lrl
```

Comment:
`copycomdb2 centos:${HOME}/db/mikedb.lrl` works

`copycomdb2 centos:~/db/mikedb.lrl` won't be expanded by the shell (at least isn't by my shell)